src/nunit.analyzers.tests/ClassicModelAssertUsage/AreEqualClassicModelAssertUsageCodeFixTests.cs

```diff
@@ -409,5 +409,89 @@ public void CodeFixUsesIsEmpty(string expected)
                 fixTitle: ClassicModelAssertUsageCodeFix.TransformToConstraintModelDescription,
                 settings: settings);
         }
+
+        [Test]
+        public void CodeFixUsesIsEmpty()
+        {
+            const string usingSystemCollectionsImmutable = "using System.Collections.Generic;";
+
+            var code = TestUtility.WrapMethodInClassNamespaceAndAddUsings($@"
+        [Test]
+        public void TestMethod()
+        {{
+            MyCollection value = new MyCollection();
+            ↓ClassicAssert.AreEqual(MyCollection.Empty, value);
+        }}
+
+        public class MyCollection : List<int>
+        {{
+            public static readonly MyCollection Empty = new MyCollection();
+        }}",
+            usingSystemCollectionsImmutable);
+
+            var fixedCode = TestUtility.WrapMethodInClassNamespaceAndAddUsings($@"
+        [Test]
+        public void TestMethod()
+        {{
+            MyCollection value = new MyCollection();
+            Assert.That(value, Is.Empty);
+        }}
+
+        public class MyCollection : List<int>
+        {{
+            public static readonly MyCollection Empty = new MyCollection();
+        }}",
+            usingSystemCollectionsImmutable);
+
+            IEnumerable<MetadataReference> existingReferences = Settings.Default.MetadataReferences ?? Enumerable.Empty<MetadataReference>();
+
+            Settings settings = Settings.Default
+                                        .WithMetadataReferences(existingReferences.Concat(MetadataReferences.Transitive(typeof(ImmutableArray<>))));
+
```

Comment:
Is this necessary (and the same comment to `settings: settings` below ?

Reply:
Thanks @mikkelbu. No it isn't. Cut&Paste from the method above and I forgot to clean up.